brave-tests/src/main/java/brave/test/util/ClassLoaders.java

```diff
@@ -94,23 +95,17 @@ public static void assertRunIsUnloadable(Class<? extends Runnable> runnable, Cla
       throw new AssertionError(e);
     }
 
-    blockOnGC();
+    try {
+      GarbageCollectors.blockOnGC(loader);
+    } catch (ConditionTimeoutException t) {
```

Comment:
this exception bites me many times which is one of the reasons I dislike awaitility. it crashes the test instead of failing it.